hack/istio/multicluster/deploy-kiali.sh

```diff
@@ -70,26 +76,36 @@ deploy_kiali() {
       echo "Pushing the images into the cluster..."
       make -e DORP="${DORP}" -e CLUSTER_TYPE="kind" -e KIND_NAME="${cluster_name}" CLUSTER_REPO=localhost cluster-push-kiali
       helm_args+=('--set deployment.image_pull_policy="Never"')
+      helm_args+=("--set deployment.image_name=localhost/kiali/kiali --set deployment.image_version=dev")
+    elif [ "${KIALI_AUTH_STRATEGY}" == "openshift" ]; then
+      local image_name
+      image_name="$(oc get image.config.openshift.io/cluster -o custom-columns=INT:.status.internalRegistryHostname --no-headers 2>/dev/null)/kiali/kiali"
+      make -e DORP="${DORP}" CLUSTER_TYPE="openshift" build build-ui cluster-push-kiali
```

Comment:
To avoid what happened earlier, make sure this script does a "cd" to the expected location thus allowing people to run this script from anywhere. Line 10 sets the SCRIPT_DIR but I do not see it "cd"'ing anywhere so this script only assumes it knows what directory it is in, but it may not be where this script thinks it is.